test-js/unit/pentaho/visual/base/View.Spec.js

```diff
@@ -0,0 +1,56 @@
+define([
+  "pentaho/visual/base/View"
+], function(View) {
+  "use strict";
+
+  describe("pentaho/visual/base/View", function() {
+    var element, dataSpec;
+
+    beforeEach(function() {
+      element = document.createElement("div");
+      var data = {};
+      dataSpec = {v: data};
+    });
+
+    describe("the constructor ", function() {
+
+      it("should throw if invoked with less than two arguments", function() {
+        expect(function() {
+          return new View();
+        }).toThrowError("Argument required: 'element'.");
+
+        expect(function() {
+          return new View(element);
+        }).toThrowError("Argument required: 'model'.");
+      });
+
+      it("should throw if the first argument is not a DOM element", function() {
+        [
+          "div", 1, true, {}, []
+        ].forEach(function(elem) {
+          expect(function() {
+            return new View(elem, dataSpec);
```

Comment:
I hadn't seen your response to my comment on the initial commit. I've answered there now. Here's what I wrote there:

> So I might not have been clear here. What I meant is that you should not pass something other than a model object into the second argument: new Model().

This applies to all `dataSpec` values being passed to argument `model` in this file.